readyapi/build.gradle

```diff
@@ -1,6 +1,6 @@
 dependencies {
     compile project(':soapui')
-    compileOnly 'com.smartbear:ready-api-soapui-pro:2.3.0'
+    compileOnly 'com.smartbear.soapui:soapui:5.4.0'
```

Comment:
The same as for: https://github.com/reportportal/agent-java-soapui/pull/14/files#diff-78e2e48110a5f79a8d5e308fa7a8cf23R3

Reply:
2.6.0 version for ready-api dependency set